src/main/java/in/gov/cdc/cdcdemostage/validators/specs/IValidator.java

```diff
@@ -0,0 +1,16 @@
+package in.gov.cdc.cdcdemostage.validators.specs;
+
+import in.gov.cdc.cdcdemostage.models.ExtractedPacket;
+import in.gov.cdc.cdcdemostage.models.ValidationError;
+
+import java.util.BitSet;
+import java.util.List;
+import java.util.Optional;
+
+public interface IValidator {
+    boolean supports(BitSet b);
+
+    String mode(String mode);
```

Comment:
Is this used to check which modes are supported by current validator. If so we can update supports signature `supports(Bitset b, String mode)`
Let me know otherwise.

Reply:
Thinking of keeping them separate as some of the validations are not dependent on mode.

Reply:
Ok. We can set Mode.ANY instead of setting string to null for validators which are not dependent on mode. `ANY` can be added to enum as mentioned here https://github.com/Vajra456/cdc-demo-stage/pull/1#discussion_r1424431665